lib/diagnostic.js

```diff
@@ -36,7 +57,15 @@ const newArray = function newArray(length, defaultValue) {
 // The function you write should return
 // the object passed in as an argument or the object created
 const addMethod = function addMethod(propertyName, method, obj) {
-
+if(obj[propertyName]) {
+    obj[propertyName] = method;
+    return obj;
+  } else {
+    obj {
```

Comment:
There is a semicolon missing here. Due to this, `grunt test` won't run. Please use `grunt nag`, like the instructions state, to correct and semantic errors. This will allow `grunt test` to run correctly and identify what is passing and what is not.